samples/sensor/bh1750/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: BH1750 ambient light sensor sample
+tests:
+  sample.sensor.bh1750:
+    harness: sensor
+    tags: sensors
+    depends_on: i2c
+    platform_allow: nrf52dk_nrf52832
+    integration_platforms:
+      - nrf52dk_nrf52832
```

Comment:
please filter by dt_compat_status enabled, this sample is not specific to nrf platforms.